test/Benchmarks/Instrumentation/UnInstrumentedHttpClientBenchmark.cs

```diff
@@ -0,0 +1,62 @@
+﻿// <copyright file="UnInstrumentedHttpClientBenchmark.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Attributes;
+using OpenTelemetry.Tests;
+
+namespace Benchmarks.Instrumentation
+{
+    [MemoryDiagnoser]
+    public class UnInstrumentedHttpClientBenchmark
```

Comment:
I'm not good at naming things. Curious to know if folks think we should call it `UninstrumentedSomething`.
It reminded me of an old joke where a C# developer asked a question in a C++ email group about "unmanaged code", immediately lots of C++ developers replied "don't call it unmanaged, use the word `native`".

Reply:
For now, I changed to `UninstrumentedSomething`.